tests/integration/suite/test_auth_configs.py

```diff
@@ -110,3 +111,12 @@ def key_secret_creation(k8sclient):
         if s.metadata.name == "pingconfig-spkey":
             return True
     return False
+
+
+def test_auth_label():
+    r = requests.post(AUTH_URL, json={
```

Comment:
Let's not do this again, one it's hardcoding a password (again) and two the token most likely already exists so if you run this test against an existing setup it can fail. Create a new user and fetch their token explicitly.

Reply:
Fixed now. Also we aren't going to backfill existing tokens.